packages/template-retail-react-app/app/components/social-login/index.jsx

```diff
@@ -43,6 +43,15 @@ const SocialLogin = ({idps}) => {
         idps && (
             <Stack spacing={4}>
                 {idps.map((name) => {
+                    if (!(name in IDP_CONFIG)) {
+                        logger.error(
+                            'IDP "' +
+                                name +
+                                '" is missing from IDP_CONFIG. Valid IDPs are [' +
+                                Object.keys(IDP_CONFIG).join(', ') +
+                                '].'
+                        )
+                    }
```

Comment:
You might want to move this logging somewhere that won't be effected by the number of renders the component has. 

If `idps` is a prop that is passed into this component you could consider doing this in a useEffect that only executes on component mount. That way the logger won't have a ton of entries as various components cause re-renders and more logging.